pkgs/tools/security/bitw/default.nix

```diff
@@ -0,0 +1,25 @@
+{ fetchFromGitHub, buildGoModule, nixosTests, lib }:
+buildGoModule rec {
+  pname = "bitw";
+  version = "unstable-2023-04-08";
+
+  vendorHash = "sha256-L6kHDZt0+QMC9BCBgh0CMyfD1lCb8ymq1sl4QqoCGH0=";
+
+  # Requires connection to bitwarden.com.
+  doCheck = false;
+
+  src = fetchFromGitHub {
+    owner = "mvdan";
+    repo = "bitw";
+    rev = "b6162395999b253c7ddff7d3a40b380a4a373ec7";
+    sha256 = "0fglyw5gidybzyjlqi255yqrgdx1dp1a4jb5in4gfa9p87gd6nvm";
+  };
+
+  passthru.tests = [ nixosTests.vaultwarden ];
+  meta = with lib; {
+    homepage = "https://github.com/mvdan/bitw";
+    description = "Bitwarden client implementing the freedesktop secrets API";
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ lheckemann ma27 hexa raitobezarius ];
```

Comment:
I signed up for this when I was actively considering to switch to bitwarden anyways soonish. Recently I decided against switching my entire setup for it, so I won't have a use for this software.

Hence, I don't think it makes sense to keep me listed here.

Reply:
> Recently I decided against switching my entire setup for it

May I ask what you decided to stick with? :)

Reply:
The very short summary is passage/age-plugin-yubikey. But... it's complicated and I wouldn't recommend that as a bit/vaultwarden replacement. Anyways, we're getting off-topic here. If you're curious, feel free to ping me :D